buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -169,19 +169,23 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
                                     }
 
                                     // Clean up defunct pipelines workspaces
-                                    dir("${buildWorkspace}/../") {
-                                        def retStatus = 0
+                                    def retStatus = 0
+                                    def cleanWSDirs = get_other_workspaces("${buildWorkspace}/../")
+
+                                    if (cleanWSDirs) {
+                                        def cleanWSDirsStr = "${buildWorkspace}/../"
```

Comment:
Can `buildWorkspace` ever be a single directory above /?  ie: `/foo/` which would become `/foo/../`

I'm worried about accidentally deleting the root of the filesystem

Reply:
buildWorkspace is set to be a directory in the user home directory: `$HOME/workspace/<job>`.
The root of the filesystem isn't (and shouldn't be) owned by jenkins user, thus the the `rm` would fail with permission denied.